eve/main.yml

```diff
@@ -456,6 +463,10 @@ stages:
         <<: *configure_end2end_tests
         env:
           <<: [*kind-env, *e2e-env, *https-env, *secrets-env]
+    - ShellCommand:
+        <<: *run_iam_policies_test
+        env:
+          <<: [ *kind-env, *e2e-env, *http-env, *secrets-env ]
```

Comment:
As discussed, we don't need to run these tests in both http and https, so we can remove that, right?

Reply:
(if we want to keep it, `http-env` should be `https-env`)

Reply:
yes yes, we dont' need it in https, I should rebase killian's branch

Reply:
I meant everywhere, see https://github.com/scality/Zenko/pull/1491/files#diff-1ef5e9752af8ccd4a1728ca19fc689a71d985bd43bb5ba2383e3c77e25e592a8R286
if Killian's PR has the linter, you won't pass it :slightly_smiling_face:

Reply:
I rebased Killian's pr, and it passed the linter surprisingly...
but I will change here anyway